### PR TITLE
Disallow raw callcode (was deprecated in 0.4.12)

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@chriseth since this practically removes the possibility of using callcode, should we also remove it from `CallKind` on the `ExpressionCompiler`?

File: Changelog.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Breaking Changes:
  * Commandline interface: Require ``-`` if standard input is used as source.
  * General: ``continue`` in a ``do...while`` loop jumps to the condition (it used to jump to the loop body). Warning: this may silently change the semantics of existing code.
  * General: Disallow declaring empty structs.
+ * General: Disallow raw ``callcode`` (was already deprecated in 0.4.12).
```
Comment (chriseth) [inline Changelog.md]: Please clarify that that this is just about the address member and not the assembly opcode.

**Review by chriseth — CHANGES_REQUESTED** [review]
Fine apart from the changelog entry. Yes, we can remove the kind enum values, but please as part of a separate PR.

**Comment by axic** [comment]
Updated.

**Review by chriseth — APPROVED** [review]

**Pull request merged**